spec/customer_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'customer'
+
+describe Customer do
```

Comment:
These tests use a real `Order` in the `Customer` class. How could you use dependency injection to isolate these unit tests from the `Order` class?